### PR TITLE
JAPI-115 Check for null returns in DFUFileDetailInfo toString method

### PR DESCRIPTION
**Pull request by Michael-Gardner**
Signed-off-by: Michael Gardner <michael.gardner@lexisnexisrisk.com>

## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).

## Checklist:
- [ ] I have created a corresponding JIRA ticket for this submission
- [x] My code follows the code style of this project.
  - [ ] I have applied the Eclipse code-format template provided.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the HPCC Systems CONTRIBUTORS document (https://github.com/hpcc-systems/HPCC-Platform/wiki/Guide-for-contributors).
- [x] The change has been fully tested:
  - [ ] I have performed unit tests to cover my changes.
  - [ ] I have performed system test and covered possible regressions and side effects.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.

### CODE REVIEW

**Comment by Michael-Gardner** [comment]
@rpastrana Please take a quick look

File: wsclient/src/main/java/org/hpccsystems/ws/client/platform/DFUFileDetailInfo.java [hunk 1]
```diff
@@ -86,52 +86,96 @@ public void setFirstline(String firstline)
...
+            sb.append("Subfiles:").append(this.getSubfiles()).append("\n");
+        if (this.getSuperfiles() != null)
+            sb.append("Superfiles:").append(this.getSuperfiles()).append("\n");
+        if (this.getUserPermission() != null)
+            sb.append("UserPermission:").append(this.getUserPermission()).append("\n");
+        if (this.getWuid() != null)
+            sb.append("Wuid:").append(this.getWuid()).append("\n");
+        if (this.getZipFile() != null)
+            sb.append("ZipFile:").append(this.getZipFile()).append("\n");
+        if (this.getColumns() != null) {
```
Comment (rpastrana) [inline wsclient/src/main/java/org/hpccsystems/ws/client/platform/DFUFileDetailInfo.java]: minor, the open curly should be in new line

File: wsclient/src/main/java/org/hpccsystems/ws/client/platform/DFUFileDetailInfo.java [hunk 2]
```diff
@@ -86,52 +86,96 @@ public void setFirstline(String firstline)
...
+            sb.append("Columns:");
+            for (DFUDataColumnInfo col : this.getColumns())
+            {
+                sb.append("    ").append(col.getColumnLabel()).append(":").append(col.toString());
```
Comment (rpastrana) [inline wsclient/src/main/java/org/hpccsystems/ws/client/platform/DFUFileDetailInfo.java]: pre-existing, but col.tostring already prints out the columlabel, so we can get rid of the .append(col.getColumnLabel()).append(":")

**Comment by Michael-Gardner** [comment]
@rpastrana Made the two changes you wanted.  Please take a quick look

**Pull request merged**